src/engine/Match_tainting_mode.ml

```diff
@@ -681,8 +681,18 @@ let check_fundef lang options taint_config opt_ent ctx java_props_cache fdef =
                     add_to_env env id ii None
                 | G.F _ -> env)
               env fields
+        | G.ParamPattern pat ->
```

Comment:
Need to refactor to avoid this duplication, but in Pro there is also a place in `Deep_tainting` where you would need to add support for `ParamPattern` (where we assign polymorphic taint).